### ISSUE TITLE
Showing limited number of days + event spanning more than 1 day shows extra days

### ISSUE THREAD

**Issue by michaeldvinci** [ljmerza/calendar-card#24]
Hello hello, it's me again! lol. [work always says I'm good at finding bugs.]

So i have a limited number of days showing on my calendar card [2] and an event spanning 4 days. it'll show all 4 days of the event, regardless of the limited number i have showing. example:

As you can see 'Off work' is a 4 day event, but I have 'numberOfDays: 2' in my config -- yet it still shows 4 days worth fo events. is there any way to hide the extra events?

```
- cards:
          - entities:
              - calendar.bills
              - calendar.events
              - calendar.gwu
              - calendar.gym_time
              - calendar.http_followshows_com_ical_9oejn9ol
              - calendar.killshot_burgundy
              - calendar.michaeldvinci_gmail_com
              - calendar.miro
              - calendar.potential_trips
              - calendar.pre_order
              - calendar.things_to_remember
              - calendar.work_solers
            numberOfDays: 2
            progressBar: true
            showMultiDay: true
            startFromToday: true
            timeFormat: 'hh:mm a'
            title: false
            type: 'custom:calendar-card'
```

![image](https://user-images.githubusercontent.com/8780953/56472070-1a0c6480-6428-11e9-83ef-4b061cd8e25d.png)

**Comment by ljmerza** [comment]
I never though of that. I'll work on it sometime this week.

**Comment by ljmerza** [comment]
fixed in v3.10.1

**Comment by michaeldvinci** [comment]
looking good!
